assets/src/js/header.js

```diff
@@ -227,5 +227,38 @@ export const setupHeader = () => {
     }
   });
 
+  /**
+   * Set the submenu behavior.
+   * Desktop should render drop/down list
+   * Mobile should render an accordion list
+   */
+  const setupSubmenu = () => {
+    document.querySelectorAll('.nav-submenu').forEach(submenu => {
+      // Find its parent menu item
+      const menuItem = document.querySelector(`*[data-menu="${submenu.dataset.parentMenu}"]`);
+
+      submenu.addEventListener('mouseover', evt => {
+        evt.preventDefault();
+
+        if(menuItem && !menuItem.classList.contains('active')) {
```

Comment:
I think it would be best to avoid adding the `active` class, since it should be reserved for the page the user is actually currently on... I think it's doable in CSS using something like:

```
.nav-item:hover a.nav-link:before {
    transform: scaleX(1);
    transition: transform 0.25s;
    border-bottom-color: var(--nav-link--text-decoration-color, $gp-green);
}
```

Reply:
If we add your suggested css it would be only applied when the nav item is hovered. Let me know if I'm correct

Reply:
The `nav-item` element is the "root" list item in the navbar, so the submenu is technically part of it, therefore hovering on the submenu is also hovering on the `nav-item`... Does that make sense? 😅